corehq/apps/api/resources/v0_5.py

```diff
@@ -1254,6 +1258,33 @@ def _date_consumer(key, value):
 
         return _date_consumer
 
+    @staticmethod
+    def _source_filter_consumer(key, value):
+        slug_values = {v: k for k, v in MessagingEvent.SOURCE_SLUGS.items()}
+        if key != "source":
+            return
+
+        if ',' in value:
```

Comment:
Isn't this whole if/else equivalent to `values = value.split(',')`?

Reply:
5eff815550a